### PR TITLE
[bridge] Fund L2 user github action

### PR DESCRIPTION
**Pull request by GuillermoLunar7**
Jira: PTP-12941

### CODE REVIEW

File: .github/workflows/fund_user.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    steps:
+      - uses: actions/checkout@v3
+      - name: Print the input tag to STDOUT
+        run: echo  Transferring ${{ inputs.amount }} to ${{ inputs.account_address }}
+      - name: Set up Python 3
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+          cache: pip
+          cache-dependency-path: solidity/paradex/contracts/cairo/requirements.txt
```
Comment (gcangussu) [inline .github/workflows/fund_user.yml]: This path doesn't exist. Did you open the PR on the wrong repository? Should it be on mono repository?

If it should be on mono, then it should be
```suggestion
          cache-dependency-path: api/paradex/contracts/cairo/requirements.txt
```

Also changing some `working-directory` in this file

Comment (gcangussu) [reply .github/workflows/fund_user.yml]: Or maybe you want to point to `cairo/requirements.txt`? Or `solidity/requirements.txt`?

Comment (GuillermoLunar7) [reply .github/workflows/fund_user.yml]: nope wrong path 100%

Comment (GuillermoLunar7) [reply .github/workflows/fund_user.yml]: it should be on the bridge repo, as it's where the dependencies are

File: .github/workflows/fund_user.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        type: string
+
+env:
+  PYTHON_VERSION: "3.9"
+  PARACLEAR_L1_ADMIN_PRIVATE_KEY: ${{ secrets.PARACLEAR_L1_ADMIN_KEY }}
+  PARACLEAR_L1_BRIDGE_ADDRESS: "0xBC5BF8f315B9D1Eebd8062CAdf113A05c5abB43E"
+  PARACLEAR_L1_TOKEN_ADDRESS: "0x66aee3c05916df7361197c0ffed6a0fdeb0b2b09b3a9bab48ac19505b82fcd"
+  WEB3_INFURA_PROJECT_ID: ${{ secrets.PARACLEAR_L1_BRIDGE_INFURA_API_KEY }}
+  AMOUNT: ${{ github.event.inputs.amount }}
+  PARACLEAR_L2_USER_ADDRESS: ${{ github.event.inputs.l2_address }}
```
Comment (gcangussu) [inline .github/workflows/fund_user.yml:22]: Or maybe we want to rename input `account_address` to `l2_address`?

```suggestion
  PARACLEAR_L2_USER_ADDRESS: ${{ github.event.inputs.account_address }}
```

**Review by gcangussu — COMMENTED** [review]
Seems like the `account_address` set on input won't be used on the python script

File: .github/workflows/fund_user.yml (hunk 2 above)
Comment (GuillermoLunar7) [reply .github/workflows/fund_user.yml:22]: we want to rename the input yes!

File: .github/workflows/fund_user.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+
+jobs:
+  fake-usdc-transfer:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Print the input tag to STDOUT
+        run: echo  Transferring ${{ inputs.amount }} USDC to L2 Address${{ inputs.l2_address }}
```
Comment (gcangussu) [inline .github/workflows/fund_user.yml]:
```suggestion
        run: echo  Transferring ${{ inputs.amount }} USDC to L2 Address ${{ inputs.l2_address }}
```

**Review by gcangussu — APPROVED** [review]
LGTM 👍

**Pull request merged**